### ISSUE TITLE
Add native support to chromatic harmonica as a plugin

### ISSUE THREAD

**Issue by GabrielReyes136** [LMMS/lmms#6071]
TLDR; The "chromatic" harmonica is very useful for improvising piano solos using a MIDI channel as an input source.

### Enhancement Summary
I'm using LMMS as a MIDI sound processor to play along Pat Metheny Groups single "Are you going with me" with my MIDI Piano using this patch.
https://lmms.io/lsp/?action=show&file=8190
ps. the chromatic harmonica is a patch on the Roland GR-55 ( i don't actually know what it means by "chromatic" but it certainly seems to add a twist to the sound).

the improvisation sounds a lot like this. 
https://youtu.be/a8oZwPGYqf8?t=209

#### Justification
The plugin posted above is a step towards using LMMS as a sound processor for MIDI instruments for more sophisticated playback of sound.

LMMS also has built in recording, which allows MIDI solos that are improvised on the keyboard to be easily recorded in high quality.

The chromatic harmonica patch is a great extension that pairs very well with MIDI input in genral.

**Comment by Spekular** [comment]
Unless I'm misunderstanding the issue description, you're using a preset from the LSP and you think it sounds good? I'm not sure how this is an enhancement request, everything is working as intended. Synths can load/save presets, and those presets can be shared on the LSP.

**Comment by GabrielReyes136** [comment]
> Unless I'm misunderstanding the issue description, you're using a preset from the LSP and you think it sounds good? I'm not sure how this is an enhancement request, everything is working as intended. Synths can load/save presets, and those presets can be shared on the LSP.

it looks like i forgot to mention i would like a chromatic harmonica to be support natively.
The issue being patches made by users often don't share the same level of quality you would expect from a developer.


and the reasoning for it being useful in the final build being having a very nice tone to test out MIDI playing real time, and also being a step towards using LMMS as a sort of piano effects processor for recording purposes
